packages/sdk/package.json

```diff
@@ -24,7 +24,7 @@
   },
   "dependencies": {
     "@rpch/common": "0.2.3",
-    "cross-fetch": "3.2.0-alpha.2",
+    "cross-fetch": "4.0.0",
```

Comment:
I think we should be able to remove this dependency, since node v18 comes with fetch built-in

Reply:
It does and cross-fetch will use it, am testing today if it works without, not just in nodejs but in all environments we run the SDK, most importantly service workers which extensions use

Reply:
we don't have programmatic tests for service workers